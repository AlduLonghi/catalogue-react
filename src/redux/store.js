import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from './reducers/index';

const initialState = {};

const store = createStore(rootReducer, initialState, applyMiddleware(thunk));

export default store;
