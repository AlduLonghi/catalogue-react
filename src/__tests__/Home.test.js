import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/home';
import { Provider } from 'react-redux';
import store from '../redux/store';

it('Snapshot test for Calculator component', () => {
  const tree = renderer
    .create(<Provider store={store}><Home /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});