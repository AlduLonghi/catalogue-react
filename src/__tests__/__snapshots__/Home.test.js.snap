// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Calculator component 1`] = `
Array [
  <header
    className="d-flex justify-content-between align-items-center"
  >
    <div
      className="d-flex align-items-center"
    >
      <img
        alt="logo"
        className="logo"
        src="gallery.svg"
      />
      <p
        data-testid="website_name"
      >
        Online Museum
      </p>
    </div>
    <select
      className="form-select"
      onChange={[Function]}
    >
      <option
        value="All"
      >
        All
      </option>
      <option
        value="Painting"
      >
        Painting
      </option>
      <option
        value="Metalwork"
      >
        Metalwork
      </option>
      <option
        value="Sculpture"
      >
        Sculpture
      </option>
      <option
        value="Ceramic"
      >
        Ceramic
      </option>
      <option
        value="Textile"
      >
        Textile
      </option>
      <option
        value="Miscellaneous"
      >
        Miscellaneous
      </option>
    </select>
  </header>,
  <main
    className="d-flex justify-content-center main-padding"
  >
    <div
      className="row artworks-col-cont"
    >
      <div
        className="col-md-4 col-12"
      />
      <div
        className="col-md-4 col-12"
      />
      <div
        className="col-md-4 col-12"
      />
      <button
        className="btn btn-warning btn-lg my-4 mx-auto"
        onClick={[Function]}
        type="button"
      >
        Load more
      </button>
    </div>
  </main>,
]
`;
